src/libs/localization-api/localization.context.ts

```diff
@@ -0,0 +1,64 @@
+import { UmbTranslationRegistry } from './registry/translation.registry.js';
+import { UmbContextToken } from '@umbraco-cms/backoffice/context-api';
+import { UmbBackofficeExtensionRegistry } from '@umbraco-cms/backoffice/extension-registry';
+import { combineLatest, distinctUntilChanged, type Observable, map } from '@umbraco-cms/backoffice/external/rxjs';
+
+export class UmbLocalizationContext {
+	#translationRegistry;
+
+	constructor(umbExtensionRegistry: UmbBackofficeExtensionRegistry) {
+		this.#translationRegistry = new UmbTranslationRegistry(umbExtensionRegistry);
+	}
+
+	get translations() {
+		return this.#translationRegistry.translations;
+	}
+
+	/**
+	 * Set a new language which will query the manifests for translations and register them.
+	 * Eventually it will update all codes visible on the screen.
+	 *
+	 * @param languageIsoCode The language to use (example: 'en-us')
+	 * @param fallbackLanguageIsoCode The fallback language to use (example: 'en-us', default: 'en-us')
+	 */
+	setLanguage(languageIsoCode: string, fallbackLanguageIsoCode?: string) {
+		this.#translationRegistry.register(languageIsoCode, fallbackLanguageIsoCode);
+	}
+
+	/**
+	 * Localize a key.
+	 * If the key is not found, the fallback is returned.
+	 * If the fallback is not provided, the key is returned.
```

Comment:
The code below returns an empty string instead of the key, so the documentation should be updated to:
```suggestion
	 * If the fallback is not provided, an empty string is returned.
```

Same for the `localizeMany` documentation...